src/app/body/tasks/tasks.component.ts

```diff
@@ -91,6 +97,23 @@ export class TasksComponent implements OnInit {
     this.router.navigate(['/Board']);
   }
 
+  changeSprint(newSprintNumber: number) {
+    if (newSprintNumber == 0) {
+      this.applicationSettingsService.getTeamDetails(this.teamId).subscribe(teams => {
+        this.teamData = teams;
+        newSprintNumber = this.teamData[0].CurrentSprintId;
+        this.currentSprintName = this.fullSprintName(newSprintNumber);
+        this.router.navigate(['Tasks/', this.teamId, this.currentSprintName]);
+        this.readData();
+      });
+    } else {
+      this.currentSprintNumber = newSprintNumber;
+      this.currentSprintName = this.fullSprintName(newSprintNumber);
```

Comment:
I don't think we are using backlog in link when somebody goes to backlog and same with deleted. So better we keep the number only